package.json

```diff
@@ -47,17 +47,19 @@
     "form-data": "^4.0.0",
     "gray-matter": "^4.0.1",
     "ignore": "^5.2.0",
-    "jsonpath": "^1.1.1",
+    "jsonpath-plus": "^7.2.0",
```

Comment:
I swapped this out because I'm using `jsonpath-plus` in `oas` for this new `analyzer` library and it doesn't make sense to have rdme ship both in users' `node_modules/` directories.

Why did I use `jsonpath-plus` instead of [jsonpath](https://www.npmjs.com/package/jsonpath)? [jsonpath-plus](https://www.npmjs.com/package/jsonpath-plus) has a nicer API than, and a larger install base and has been recently updated within the past few months vs 2 years.